Makefile

```diff
@@ -1,55 +1,50 @@
-VIRTUALENV = virtualenv
 VENV := $(shell echo $${VIRTUAL_ENV-$$PWD/.venv})
 PYTHON = $(VENV)/bin/python
-DEV_STAMP = $(VENV)/.dev_env_installed.stamp
 INSTALL_STAMP = $(VENV)/.install.stamp
-TEMPDIR := $(shell mktemp -d)
 
 .IGNORE: clean
-.PHONY: all install virtualenv tests install-dev tests-once
+.PHONY: all install virtualenv tests tests-once
 
 OBJECTS = .venv .coverage
 
 all: install
-install: $(INSTALL_STAMP)
-$(INSTALL_STAMP): $(PYTHON) requirements.txt setup.py
+
+install: $(INSTALL_STAMP) pyproject.toml requirements.txt
+$(INSTALL_STAMP): $(PYTHON) pyproject.toml requirements.txt
 	$(VENV)/bin/pip install -r requirements.txt
-	$(PYTHON) setup.py develop
+	$(VENV)/bin/pip install ".[dev]"
 	touch $(INSTALL_STAMP)
 
-install-dev: $(INSTALL_STAMP) $(DEV_STAMP)
-$(DEV_STAMP): $(PYTHON) dev-requirements.txt
-	$(VENV)/bin/pip install -r dev-requirements.txt
-	touch $(DEV_STAMP)
-
 virtualenv: $(PYTHON)
 $(PYTHON):
-	virtualenv $(VENV)
+	python -m venv $(VENV)
```

Comment:
Is this worth having as a command on its own? Or should it be done automatically as a dependency of `install`?